### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by Outkarm**
- add HTML layout and tags to setup
- add the required style to mimic wireframe guide
- add HTML webpack config

### CODE REVIEW

**Comment by DammyShittu** [comment]
Hi @Outkarm,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a project that does not have a descriptive readme doc. Use the template for your readme.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,101 +1,136 @@
...
+    HTML
+    CSS
+    Javascript(ES6)
+    Webpack
 
-An open-source, automated tool for improving the quality of web pages. It has audits for performance, accessibility, progressive web apps, SEO and more.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-You can get the Lighthouse report by any of the following ways:
+## 🚀 Live Demo <a name="live-demo" href='https://outkarm.github.io/To-Do-List-App/dist/'>Click Here to view</a>
```
Comment (1GORDON) [inline README.md:42]: Great job @Outkarm 🤝

- [ ] [Optional] Kindly remove this live demo link as it is for a different project.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @Outkarm ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Outkarm) [reply README.md:42]: @1GORDON  thanks for the tip, but I will put the demo link when i am done with the whole project

**Pull request merged**